lib/rouge/lexers/viml.rb

```diff
@@ -13,10 +13,7 @@ class VimL < RegexLexer
 
       mimetypes 'text/x-vim'
 
-      def self.keywords
-        Kernel::load File.join(Lexers::BASE_DIR, 'viml/keywords.rb')
-        self.keywords
-      end
+      require_relative "viml/keywords"
```

Comment:
Hm. The reason we did it this way was that these giant keyword files have a massive memory footprint, and we don't want Rouge to load them unless the lexer is actually in use. I think it'll still work if we use `require_relative` dynamically maybe?

Reply:
running the above benchmark code with `/usr/bin/time -lph`:

```
master:

            61194240  maximum resident set size
                3888  page reclaims
                   4  page faults
                  17  involuntary context switches
          6141564240  instructions retired
          1349472140  cycles elapsed
            55493952  peak memory footprint

mine:

            59604992  maximum resident set size
                3791  page reclaims
                   4  page faults
                  10  involuntary context switches
          6451778027  instructions retired
          1398372284  cycles elapsed
            51660096  peak memory footprint
```
(removed lines reporting 0)

I reran each 3 times and the numbers are all consistent...

Reply:
cc @ashmaroli